hack/bin/create_test_team_admins.sh

```diff
@@ -12,6 +12,9 @@ USAGE="
 This bash script can be used to create active team admin users and
 their teams.
 
+This is the way to create teams if you have set
+'setRestrictUserCreation' to false in your 'values.yaml'.
```

Comment:
should this not be "setRestrictUserCreation" to true - or am I confused here?

Reply:
nope. you're right, thanks!  aea091d255423880d37731466b0b7f2d2f8bed0d